playbooks/code-review.md

```diff
@@ -2,3 +2,45 @@
 title: Code reviews
 description: How our code review process works 
 ---
+
+Reviewing other people's code is a responsibility of every single developer. Instead of viewing it as a hassle, it should be looked upon as a necessary process in order to keep the code clean and consistent.
+It is also an opportunity to learn something new.
+
+Rules on..
+
+## How a Code Review process should look like
+
+..are written below.
+
+### Terms:
+ - PR - Pull Request.
+ - PR Creator - a person who initiated the PR.
+ - Reviewer - a person assigned to do a PR code review.
+
+### Rules:
+1. Every PR has to be reviewed, with the exception of fix/hotfix PR-s that can be merged by the code owner on his own responsibility.
+2. When creating a PR, you should select **EXACTLY ONE** person as a reviewer. This person has to be someone who is working at the moment (not on vacation). 
```

Comment:
It is important to mention WHY it is important that only one person is a reviewer. I would say that it mitigates the problem of "who should actually look at this" or "oh there's a few of us so someone will look at it, im busy" and if you are busy that YOU as the reviewer are responsible to assign a new reviewer, and stuff like that

Reply:
I tried to avoid going too deep into why some things are the way they are. That makes the document bloated and dissuades people from reading it. This is the recipe on how a Code Review process should look like.

Reply:
I would also provide explanation to the WHY. Knowing WHY helps people to better understand and accept ideas and rules.